src/js/game/LevelMVC/LevelView.js

```diff
@@ -769,6 +782,18 @@ module.exports = class LevelView {
     }
   }
 
+  /**
+  * Handling the first case of walking onto a track while not currently on one
+  */
+  firstTime(currPos, nextPos) {
+    let nextBlock = this.controller.levelModel.actionPlane.getBlockAt(nextPos);
+    let currBlock = this.controller.levelModel.actionPlane.getBlockAt(currPos);
+    if (!currBlock.blockType.startsWith("rails") && nextBlock.blockType.startsWith("rails")) {
```

Comment:
these should both _definitely_ be using `isRails` to avoid `railsRedstoneTorch`

Reply:
isRails doesn't seem to be a thing currently? I can make a helper for that though 👍